shared/middleware/channels_api.rb

```diff
@@ -164,7 +164,7 @@ class ChannelsApi < Sinatra::Base
   # Marks the specified channel as published.
   #
   post %r{/v3/channels/([^/]+)/publish/([^/]+)} do |channel_id, project_type|
-    not_authorized if under_13?
+    not_authorized if under_13? && !%w(artist playlab).include?(project_type)
```

Comment:
> `/v3/channels/{channel}/publish/foo` doesn't seem like a valid URL

You are correct. I've added a whitelist for project type in the over-13 case, and added tests to ensure that project type `foo` is rejected.